File: NERP.NetworkEntities/Models/Chamber.cs
using NERP.Common.Models;

namespace NERP.NetworkEntities.Models
{
    public class Chamber : BaseNetworkEntity
    {
        public Point Geometry { get; set; }
        public string ChamberCategory { get; set; }
        public string ChamberClosureType { get; set; }
        public string SpliceTrayCount { get; set; }
        public bool IsBuried { get; set; }
        public Installation Installation { get; set; }
        public Owner Owner { get; set; }
        public string LoopLength { get; set; }
    }
}

File: NERP.Admin.Web/ClientApp/src/app/entities/models/chamber.ts
import { Point } from '../../common/geo/geometry';
import { BaseNetworkEntity } from './baseNetworkEntity';
import { Installation } from './installation';
import { Owner } from './owner';

export class Chamber extends BaseNetworkEntity {

   chamberCategory?: string;
   chamberClosureType?: string;
    spliceTrayCount?: number;
    isBuried?: boolean;

    geometry: Point;
    installation?: Installation = new Installation();
    owner?: Owner = new Owner();
}

